### ISSUE TITLE
The drop down menus remain open with everything else greyed out

### ISSUE THREAD

**Issue by AgelxNash** [evolution-cms/evolution#890]
The drop down menus remain open with everything else greyed out. I can change drop downs by going to a different link but I cannot get rid of a drop down. They remain persistent.

_Originally posted by @nick0 in https://github.com/evolution-cms/evolution/issues/886#issuecomment-437506100_

**Comment by nick0** [comment]
I think #890 and #889 are definitely related to #886.
The problems only occured after taking advice in that thread to try the ElementsInTree plugin from the 1.4.x archive. That's when this issue started.

All cache and history etc in all browsers are cleared after closing each time.

- Firefox 63 in privacy and normal modes - Header drop down menus can be clicked but remain persistent. Tree and Content frames are greyed out per screencap given in #886. Not working properly and original problem persists

- Opera 56 - Header menus can be clicked to reveal drop down menu. Nothing is greyed out. Working
- IE 11 - Header menus can be clicked to reveal drop down menu. Nothing is greyed out. Working.
- Chrome 70
-- in incognito mode: Header menus can be clicked to reveal drop down menu. Nothing is greyed out. Working. 
--- Although there are 23 errors shown in the console
3tabpane.js: 15 Uncaught TypeError: Cannot read property 'hasChildNodes' of undefined

Could those errors be related to the Firefox issue?

Any ideas or suggestions?

Thanks.

**Comment by AgelxNash** [comment]
@nick0 Problem solved?

**Comment by nick0** [comment]
No unfortunately, problem still exists.
Even with a manual upload / install of 1.4.6 using the same database.

**Comment by Nicola1971** [comment]
chrome extensions?
manager plugins?

**Comment by nick0** [comment]
Its in Firefox 63 in privacy and normal modes - no change with all extensions removed.
Only occured in 1.4.6 after the ElementsInTree issue I had was resolved. Something lingering I think.

**Comment by AgelxNash** [comment]
Show the list of manager plugins

**Comment by nick0** [comment]
Thanks for your help. Here you go...

Pic showing the problem:
![untitled-1](https://user-images.githubusercontent.com/1768308/48384630-2aaf0500-e73b-11e8-9330-ff1013b18e04.jpg)

Pic showing all plugins after clicking the top level link to close:
![untitled-2](https://user-images.githubusercontent.com/1768308/48384646-37cbf400-e73b-11e8-8c45-fe4ecb20aa32.jpg)

Note also: I just updated two other sites to 1.4.6 from 1.4.5 and do not have this problem in separate tabs of the same session of the same browser (Firefox 63). Nor did I have that ElementsInTree problem as I disabled before upgrading. It definitely appears localised to this one site only so I can't help thinking it was related or that something is lingering.

**Comment by Nicola1971** [comment]
Try to disable Evodashboard

**Comment by nick0** [comment]
Just tried disabling Evodashboard - no difference.

I am now up to four other 1.4.6 sites without this problem. Also those sites all have the same plugins as per the images above, including Evodashboard.

**Comment by Nicola1971** [comment]
anyone can confirm or replicate this issue?

**Comment by dmi3yy** [comment]
Cant reproduce

**Comment by nick0** [comment]
I wish this issue wasn't closed...
I am still having this issue in Firefox (normal and private browsing) and also in Chrome (normal and incognito).
Now in sites updated to Evo 1.4.8 from 1.2.1, it affects about 60% of sites I have updated.

Here is a video showing the issue:
[https://screencast-o-matic.com/watch/cqnIcR3Lyl](https://screencast-o-matic.com/watch/cqnIcR3Lyl)

On all sites that have this issue, it seems that there is a resizing of some sections (notably database backup section and elements) that happens after clicking the link. See what happens in the database section below
[https://screencast-o-matic.com/watch/cqnIcg3LwU](https://screencast-o-matic.com/watch/cqnIcg3LwU)
Notably in IE 11 that does not have the persistent menu / greyed out issue, there is no resizing happening.

Any ideas?

**Comment by 64j** [comment]
@nick0 
Try to save the configuration.

**Comment by nick0** [comment]
Thanks Serg. If you mean the main configuration (at System > Configuration), I can save that on all of the affected sites. Just tested again, saving that file makes no difference to the issue I am seeing.

**Comment by nick0** [comment]
Please reopen this item. Very much still an issue, only now in many more sites.

Any other ideas? Could there be something in the database related to #886 that perhaps needs removing? If so, what do I need to remove? 

Any other ideas?

**Comment by nick0** [comment]
Hello,

Please reopen per above.

**Comment by nick0** [comment]
Thank you for reopening.

Just confirming the latest info from the above posts...

I am still having this issue in Firefox (normal and private browsing) and also in Chrome (normal and incognito). Now in sites updated to Evo 1.4.8 from 1.2.1, it affects about 60% of sites I have updated.

Here is a video showing the issue:
[https://screencast-o-matic.com/watch/cqnIcR3Lyl](https://screencast-o-matic.com/watch/cqnIcR3Lyl)

On all sites that have this issue, it seems that there is a resizing of some sections (notably database backup section and also elements) that happens after clicking the link / tabs. See what happens in the database section below
[https://screencast-o-matic.com/watch/cqnIcg3LwU](https://screencast-o-matic.com/watch/cqnIcg3LwU)
Notably in IE 11 and Opera 57 that do not have the persistent menu / greyed out issue, there is no resizing happening.

Any ideas?

Could there perhaps be something in the database related to #886 that perhaps needs removing? If so, what do I need to look for and remove?

I am happy to provide manager access to one of the senior devs if someone is keen to look at this issue in action.

Thanks for any help.

**Comment by 64j** [comment]
@nick0 
I clicked the buttons in a chaotic manner in the browser and could not repeat your mistake.
Do not misunderstand me.
No confirmation of your error


https://monosnap.com/file/yZkWIRQ9h3zZxYMWTRzwLuqo85EKmZ

**Comment by nick0** [comment]
Thank you @64j.
What you show in your video in what I see / can do in other sites that don't have this problem.
I know that what you show is the standard behaviour and exactly what I am hoping to achieve on the affected sites. I would love them to work like normal like yours :).
If I gave you access to the manager of one of my sites that has this problem, would you be interested in checking for me? If so, how can send this info to you?
Thanks again

**Comment by nick0** [comment]
Found it! You beauty. 

It was a Cloudflare related issue.

I took a copy of the site and database and placed it on a dummy domain - no issue on the dummy and I was stumped. The only difference was Cloudflare so I had a good look there. 

A Cloudflare "disable performance" pagerule is needed to ensure Cloudflare does not stuff up the EvolutionCMS Manager. When I checked the pagerules, I noticed that the manager folder name was wrong. I had changed the manager folder name some time ago (likely just before this issue was first reported) and forgot to update it in the page rule. Doh! I fixed that and refreshed everything and no problems anymore on my main site.

It did seem like a javascript issue so this make sense. Sorry for the goose chase.

Tested on 3 other client sites with the issue and which also use Cloudflare. Fixed those too.
 
Thanks for reopening this issue and for all the help with this. 

Cheers and have a great day :)
